### PR TITLE
Add general aRMT topic

### PR DESCRIPTION
**Pull request by blootsvoets**
Adds a general `questionnaire_response` Kafka topic to the aRMT app, as well as moving the monitoring topics to that general device type. This allows an installation to have far fewer topics if the users do not require questionnaire responses to be separated over different topics. For example in Confluent Cloud this makes a big cost difference.

### CODE REVIEW

**Review by mpgxvii — APPROVED** [review]
This is a good idea. LGTM. Thanks!

File: specifications/active/aRMT-1.17.0.yml [hunk 1]
```diff
@@ -0,0 +1,299 @@
+name: aRMT
```
Comment (nivemaham) [inline specifications/active/aRMT-1.17.0.yml:1]: why do we need this new specification?

Comment (blootsvoets) [reply specifications/active/aRMT-1.17.0.yml:1]: It removes the topics that are already in the general spec. There isn’t a conflict though, so I can also remove this and just keep 1.16.0…

Comment (nivemaham) [reply specifications/active/aRMT-1.17.0.yml:1]: Should it be `questionnaire_response` instead of `notification_thinc_it` and so on in that case?

Comment (blootsvoets) [reply specifications/active/aRMT-1.17.0.yml:1]: Well, so the generic topics, including `questionnaire_response` are now in the generic specification, so I thought there would be no need to duplicate them here.

**Pull request merged**